libs/orders/frontend/view/src/lib/components/orders.ts

```diff
@@ -0,0 +1,27 @@
+export interface Order {
+	id: string;
+	icon: string;
+	notification: string;
+	heading: string;
+	subheading: string;
+	variant: string;
+}
+
+export const orders: Order[] = [
+	{
+		id: '1',
+		icon: 'battery-empty',
+		notification: '99+',
+		heading: 'Fix the Aquarium Pump',
+		subheading: 'The pump in the aquarium is not working properly',
+		variant: 'warning',
+	},
+	{
+		id: '2',
+		icon: 'water-fish',
+		notification: '3',
+		heading: 'Leaking Water Tank',
+		subheading: 'The water tank in the CS Department building is leaking',
+		variant: 'critical',
+	},
+];
```

Comment:
same as for the `facilities`.

Reply:
Will be done as part of the order form issue https://github.com/orgs/amosproj/projects/43/views/2?pane=issue&itemId=62262011